packages/flutter/lib/src/foundation/basic_types.dart

```diff
@@ -34,25 +34,48 @@ class BitField<T extends dynamic> {
   static const int _kSMIBits = 63; // see https://www.dartlang.org/articles/numeric-computation/#smis-and-mints
   static const int _kAllZeros = 0;
   static const int _kAllOnes = 0x7FFFFFFFFFFFFFFF; // 2^(_kSMIBits+1)-1
```

Comment:
This might be wrong. SMIs are signed. I think this should actually be 0x3FFFFFFFFFFFFFFF and you only get 62 bits, because one of the bits is the signed bit.
https://www.dartlang.org/articles/numeric-computation/#smis-and-mints